src/binary/binary_writer.rs

```diff
@@ -201,10 +201,10 @@ impl<W: Write> IonWriter for BinaryWriter<W> {
 #[cfg(test)]
 mod tests {
     use super::*;
+    use crate::ion_reader::IonReader;
     use crate::reader::ReaderBuilder;
-    use crate::stream_reader::IonReader;
 
-    use crate::StreamItem::Value;
+    use crate::reader::StreamItem::Value;
```

Comment:
🗺️ Now that the root-level public re-exports of the reader-related types are only available conditionally, internal modules need to refer to the internally visible path of each type.

In this example, `crate::StreamItem::Value` became `crate::reader::StreamItem::Value` because `StreamItem` is no longer guaranteed to be publicly visible at the crate root.